js/_current-weather.js

```diff
@@ -209,15 +246,17 @@ function updateBackgroundImage(weather) {
 }
 // Function to update the clock display
 function updateClock() {
-  const clockElement = document.getElementById('current-city-clock');
+  const clockElement = document.getElementById('current-city-clock-section--clock');
+  const ampmClock = document.getElementById('current-city-clock-section--ampm');
   const currentTime = new Date();
   const hours = currentTime.getHours();
   const minutes = currentTime.getMinutes();
   const ampm = hours >= 12 ? 'PM' : 'AM';
   const formattedHours = hours % 12 || 12; // Convert to 12-hour format
 
-  const formattedTime = `${formattedHours}:${padZero(minutes)} ${ampm}`;
+  const formattedTime = `${formattedHours}:${padZero(minutes)}`;
```

Comment:
I should have commented in the last PR, but here's two points to talk about zero padding function.
1. If you want to use padZero function, it should be declared in util.js as it's a utility function which can be used generally in entire application.
2. There is a built-in method called `padStart` in Javascript to realize this.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart

I personally prefer using built-in method as much as I can, but it's up to you:)